From 73559a25e0fb484fc3ba42f1796b9b2b46128714 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Thu, 4 Aug 2011 06:35:44 +0000
Subject: [PATCH] sysctl: Use /dev/null instead of -q

Busybox sysctl has no -q option so use /dev/null instead.
---
 init.d/sysctl.Linux.in |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/init.d/sysctl.Linux.in b/init.d/sysctl.Linux.in
index 71932fe..9449c5c 100644
--- a/init.d/sysctl.Linux.in
+++ b/init.d/sysctl.Linux.in
@@ -18,9 +18,9 @@ start()
 	for conf in /etc/sysctl.d/*.conf /etc/sysctl.conf; do
 		if [ -r "$conf" ]; then
 			vebegin "applying $conf"
-			if ! err=$(sysctl -q -p "$conf" 2>&1) ; then
+			if ! err=$(sysctl -p "$conf" 2>&1 >/dev/null) ; then
 				errs="${errs} ${err}"
-				sysctl -q -e -p "${conf}"
+				sysctl -e -p "${conf}" >/dev/null
 			fi
 			veend $? || retval=1
 		fi
-- 
1.7.6

